cpp/tests/types/dispatcher_test.cu

```diff
@@ -13,139 +13,126 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-
-#include <cudf.h>
+#include <tests/utilities/cudf_test_fixtures.h>
 
 #include <utilities/type_dispatcher.hpp>
-#include "gtest/gtest.h"
-#include "tests/utilities/cudf_test_fixtures.h"
+#include <cudf.h>
 
 #include <thrust/device_vector.h>
-#include <cstdint>
 
+#include <gtest/gtest.h> 
+
+#include <cstdint>
 
 /**
  * @file dispatcher_test.cu
  * @brief Tests the type_dispatcher
- */
-
-struct DispatcherTest : public GdfTest {
-  /**---------------------------------------------------------------------------*
-   * @brief Lists every gdf_dtype that the type_dispatcher supports.
-   *
-   * This vector *must* list every gdf_dtype supported by the type_dispatcher.
-   *
-   * If a new type gdf_dtype is added, but this list is not updated, then the
-   * tests will fail.
-   *
-   *---------------------------------------------------------------------------**/
-  std::vector<gdf_dtype> supported_dtypes{
-      GDF_INT8,    GDF_INT16,  GDF_INT32,  GDF_INT64,     GDF_FLOAT32,
-      GDF_FLOAT64, GDF_DATE32, GDF_DATE64, GDF_TIMESTAMP, GDF_CATEGORY};
-
-  // These types are not supported by the type_dispatcher
-  std::vector<gdf_dtype> unsupported_dtypes{GDF_invalid, GDF_STRING};
-};
-
-using TestTypes = ::testing::Types<int8_t, int16_t, int32_t, int64_t, float,
-                                   double, cudf::date32, cudf::date64,
-                                   cudf::timestamp, cudf::category>;
-
-template <typename T>
-struct TypedDispatcherTest : DispatcherTest {};
+*/
 
-TYPED_TEST_CASE(TypedDispatcherTest, TestTypes);
+namespace{ 
 
-namespace {
-template <typename ExpectedType>
-struct type_tester {
-  template <typename DispatchedType>
-  bool operator()() {
-    return std::is_same<ExpectedType, DispatchedType>::value;
+struct test_functor {
```

Comment:
This conflict resolution is incorrect, it overwrote the new `type_tester` functor.

Reply:
>  I'll review in-depth once the merge conflicts have been correctly resolved.

Are the places you specifically marked here the only incorrect merges?

git did this automagically and I didn't even touch `dispatcher_test.cu` manually. Anyway, I'll have another look at the entire diff with branch-0.6.